utility/module.go

```diff
@@ -1,35 +1,54 @@
 package utility
 
 import (
+	"errors"
+
 	"github.com/pokt-network/pocket/logger"
 	"github.com/pokt-network/pocket/runtime/configs"
 	"github.com/pokt-network/pocket/shared/mempool"
 	"github.com/pokt-network/pocket/shared/modules"
 	"github.com/pokt-network/pocket/shared/modules/base_modules"
+	"github.com/pokt-network/pocket/utility/fisherman"
+	fisherman_module "github.com/pokt-network/pocket/utility/fisherman"
```

Comment:
I think we should go with just `fisherman` instead of `fisherman_module`; ditto elsewhere

https://go.dev/blog/package-names

<img width="851" alt="Screenshot 2023-05-03 at 7 09 53 PM" src="https://user-images.githubusercontent.com/1892194/236096025-6c88a5d5-efe0-4075-a91f-36d270dae6e3.png">